nexus/pkg/server/sender.go

```diff
@@ -281,13 +439,34 @@ func (s *Sender) sendRun(record *service.Record, run *service.RunRecord) {
 		nil,                      // summaryMetrics
 	)
 	if err != nil {
-		err = fmt.Errorf("sender: sendRun: failed to upsert bucket: %s", err)
-		s.logger.CaptureFatalAndPanic("sender received error", err)
+		err = fmt.Errorf("failed to upsert bucket: %s", err)
+		s.logger.Error("sender: sendRun:", "error", err)
+		result := &service.Result{
```

Comment:
you are missing the Control and UUid fields, this wont work without them..  might introduce a result from response helper like we have in python

Reply:
no need to use blocking... i always address comments! just requires another round of reviews...

Reply:
yeah, typo...